flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/DeltaJoinTest.scala

```diff
@@ -624,6 +624,21 @@ class DeltaJoinTest extends TableTestBase {
       ExplainDetail.PLAN_ADVICE)
   }
 
+  @Test
+  def testLookupTableWithCache(): Unit = {
```

Comment:
Is there an end to end test we can add to drive this code outside of explain? If this issue is explain only ,then we should mention the situations that the user can hit this in the Jira or PR text - so people can find it easier.